mod_auth/forms.py

```diff
@@ -62,20 +62,6 @@ def _email_not_in_use(form, field):
     return _email_not_in_use
 
 
-def role_id_is_valid(form, field):
```

Comment:
Removed as it's unused?

Reply:
Yes. I've mentioned in the commit; there is no model `Role` and hence there is no way we can query it. This is breaking but was never called.